### PR TITLE
github workflows

### PR DESCRIPTION
**Pull request by priyanka-surana**
Copied changes to GitHub workflows

## PR checklist

Closes #XXX 

- [x ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: .github/workflows/pytest-workflow.yml [hunk 1]
```diff
@@ -41,6 +44,13 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Login to Github Packages
+        uses: docker/login-action@v2
```
Comment (priyanka-surana) [inline .github/workflows/pytest-workflow.yml:48]: Does this `uses` need to match the one above?

So its `uses: actions/checkout@v3`, should this line be `uses: docker/login-action@v3`? Not sure.

Comment (muffato) [reply .github/workflows/pytest-workflow.yml:48]: Not necessarily. They're essentially different software with different release schedules, so it should be fine to use v2 and v3

**Comment by muffato** [comment]
I'm curious: where did you get the `.github/workflows/pytest-workflow.yml` change from ?

**Comment by priyanka-surana** [comment]
> I'm curious: where did you get the `.github/workflows/pytest-workflow.yml` change from ?

Its exactly as it is in the current `nf-core/modules` + the changes from `.github/workflows/pytest-workflow.yml`

**Comment by muffato** [comment]
Ah right ! That's to allow running CI on our own, private, images, like longranger

**Review by muffato — APPROVED** [review]

**Pull request merged**